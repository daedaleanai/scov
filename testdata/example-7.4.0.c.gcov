file:example.c
function:28,1,main
lcount:28,1
lcount:34,1
lcount:36,1
lcount:37,1
lcount:43,1
lcount:44,1
lcount:49,1
lcount:51,0
lcount:55,1
lcount:58,1
