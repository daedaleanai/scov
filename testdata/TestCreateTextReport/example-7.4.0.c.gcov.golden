 Lines	 Funcs
------	------
 90.0%	100.0%	example.c
------	------
 90.0%	100.0%	Overall
